src/detours.cpp

```diff
@@ -1515,6 +1548,8 @@ struct DetourOperation
     ULONG               dwPerm;
 };
 
+static HANDLE				s_hHeap = NULL;
```

Comment:
Never used?

Reply:
Yes, these codes here has expired, and these code are no longer existed in my latest optimized commit.